Dockerfile

```diff
@@ -1,22 +1,19 @@
-# ubuntu 16.04 xenial image
-FROM ubuntu
+# use base python
+FROM python:2.7
 
-# apt update it
-RUN apt-get update
-# get essentials
-RUN apt-get install -y tar git curl wget dialog net-tools build-essential python python-dev python-distribute python-pip
```

Comment:
It comes with tar, git, curl, and wget.  I'm not sure why we needed dialog or net-tools here, maybe @purukaushik knows?  None of the packages in requirements need build-essential / a compiler and it severely bloats the image size.  Python and Pip are compiled / installed in the base image.

```
root@5af2fe28c5d0:/app# dpkg -l | grep "^ii  \(tar\|git\|curl\|wget\|dialog\|net-tools\) "
ii  curl                              7.38.0-4+deb8u3            amd64        command line tool for transferring data with URL syntax
ii  git                               1:2.1.4-2.1+deb8u2         amd64        fast, scalable, distributed revision control system
ii  tar                               1.27.1-2+b1                amd64        GNU version of the tar archiving utility
ii  wget                              1.16-1                     amd64        retrieves files from the web
root@5af2fe28c5d0:/app# python --version
Python 2.7.12
root@5af2fe28c5d0:/app# pip --version
pip 8.1.2 from /usr/local/lib/python2.7/site-packages (python 2.7)
```